### ISSUE TITLE
nearby(): expose minDist parameter

### ISSUE THREAD

**Issue by noerw** [public-transport/hafas-client#183]
I saw in `formatNearbyReq()`, that the API provides a `minDist` parameter.
Is there a reason this is not exposed in hafas-client?
This could be done without breaking changes, although it would probably make sense to rename the current `distance` option to `maxDistance` then.

**Comment by noerw** [comment]
I did some tests with DB HAFAS, this parameter is not intuitive at all:

1. only works when `maxDist` is also set
2. HAFAS applies distance filter **after** result count filter. This makes `minDist` mostly useless, as you often won't get any results back unless you query for a large number of results

**Comment by derhuerst** [comment]
> This could be done without breaking changes

Yes, we could do that.

> although it would probably make sense to rename the current distance option to `maxDistance` then.

That would be a breaking change.

> I did some tests with DB HAFAS, this parameter is not intuitive at all: [it] only works when `maxDist` is also set

This makes it even more unintuitive, but we could still add it. What do you think?

**Comment by derhuerst** [comment]
> Is there a better way to geographically match stops eg by prefetching them?

Yes there is! When trying to do exactly this, I came up with [`hafas-find-stations`](https://github.com/derhuerst/hafas-find-stations), which saves you from figuring out a few more unintuitive things. If you have trouble using it, please create an Issue over there.

Also, how would you change the documentation of this repo, so that others will find `hafas-find-stations` in the future?
